README.md

```diff
@@ -36,6 +36,7 @@ Simple use cases with MPESA Express (STKPush) will look something like this:
 Have a Daraja singleton instance 
 
 ```java
+ class Payment{
```

Comment:
A few comments here.

I think you can remove the updates on the readme. The reason is 
1. We want to make the sample app DI free - this way, a user can choose how best to integrate the lib on their end (Right now, everything is setup using dagger)
2. We're re-writting the daraja library, and we might choose to change a few things on how the library works - this might affect the sample codes you added on the README

Reply:
Sure on it

Reply:
Done, reverted the changes